iron-query-params.html

```diff
@@ -50,7 +50,7 @@
         return;
       }
       this.paramsString = this._encodeParams(this.paramsObject)
-          .replace(/%3F/g, '?').replace(/%2F/g, '/');
+          .replace(/%3F/g, '?').replace(/%2F/g, '/').replace("'", '%27');
```

Comment:
(Whoops, probably should have mentioned this before LGTM'ing)

This should be a global replace, not just the first quote.